k8s/netmanager/values-dev.yaml

```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: airqo-dev-platform-frontend
+  repository: gcr.io/airqo-250220/pr-previews/netmanager-pr-previews
```

Comment:
A quick one @123MwanjeMike , aren't we also able to use ENVs here?

Reply:
Hi @Baalmart 
We unfortunately can't for the kubernetes and helm files